src/Domain/FunctionsOfTime/Tags.hpp

```diff
@@ -37,26 +41,44 @@ struct OptionList {
       tmpl::list<domain::OptionTags::DomainCreator<Metavariables::volume_dim>,
                  domain::FunctionsOfTime::OptionTags::FunctionOfTimeFile,
                  domain::FunctionsOfTime::OptionTags::FunctionOfTimeNameMap>,
-      tmpl::list<domain::OptionTags::DomainCreator<Metavariables::volume_dim>>>;
+      tmpl::flatten<tmpl::list<
```

Comment:
I think this is the same as the list below, so how about `typename OptionList<Metavariables, false>::type`?